node/src/command-options/ConditionalChange.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+/**
+ *
+ * An optional condition to the GeoAdd command.
```

Comment:
You can add a link to command, but keep in mind adding an import. See #1982 for example